setup.py

```diff
@@ -175,12 +175,33 @@ def _parse_value(name: str, typed_value: str, type_hint: Optional[str] = None) -
     return VariableDefinition(type_name=type_name, value=typed_value[i+1:-1], comment=comment, type_hint=type_hint)
 
 
-def _update_constant_vars_with_kzg_setups(constant_vars, preset_name):
+def _update_constant_vars_with_kzg_setups(constant_vars, preset_dep_constant_vars, preset_name):
     comment = "noqa: E501"
     kzg_setups = ALL_KZG_SETUPS[preset_name]
-    constant_vars['KZG_SETUP_G1_MONOMIAL'] = VariableDefinition(constant_vars['KZG_SETUP_G1_MONOMIAL'].value, str(kzg_setups[0]), comment, None)
-    constant_vars['KZG_SETUP_G1_LAGRANGE'] = VariableDefinition(constant_vars['KZG_SETUP_G1_LAGRANGE'].value, str(kzg_setups[1]), comment, None)
-    constant_vars['KZG_SETUP_G2_MONOMIAL'] = VariableDefinition(constant_vars['KZG_SETUP_G2_MONOMIAL'].value, str(kzg_setups[2]), comment, None)
+    preset_dep_constant_vars['KZG_SETUP_G1_MONOMIAL'] = VariableDefinition(
+        preset_dep_constant_vars['KZG_SETUP_G1_MONOMIAL'].value,
+        str(kzg_setups[0]),
+        comment, None
+    )
+    preset_dep_constant_vars['KZG_SETUP_G1_LAGRANGE'] = VariableDefinition(
+        preset_dep_constant_vars['KZG_SETUP_G1_LAGRANGE'].value,
+        str(kzg_setups[1]),
+        comment, None
+    )
+    constant_vars['KZG_SETUP_G2_MONOMIAL'] = VariableDefinition(
+        constant_vars['KZG_SETUP_G2_MONOMIAL'].value,
+        str(kzg_setups[2]),
+        comment, None
+    )
```

Comment:
Why are the first two part of `preset_dep_constant_vars` and the third part of `constant_vars`?

Hmm because the first two have "different" values for presets. In reality these will always be the same. Maybe it would be better to fix this; make the first two like `KZG_SETUP_G2_MONOMIAL`.

Reply:
They get assigned into the two groups depending on whether their computed value depends on preset variables. I'm just changing the value in the list where they end up getting assigned. The only difference of `preset_dep_constant_vars` vs `constant_vars` is the order in which they are emitted. `preset_dep_constant_vars` comes after preset vars, `constant_vars` comes before (before some presets depend on constants)

Reply:
<img width="696" alt="image" src="https://github.com/user-attachments/assets/59391abc-a517-4e69-96a6-8d7f435cd384" />

Reply:
I see. It's because these first two depend on the `FIELD_ELEMENTS_PER_BLOB` preset whereas `KZG_SETUP_G2_MONOMIAL` does not; its size is a 65 regardless of anything else.